### PR TITLE
Point BigQuery docs to docs index

### PR DESCRIPTION
**Pull request by tswast**

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
It looks like the various services are inconsistent - could you make sure they all point to the index?

**Comment by tswast** [comment]
Sure, I'll update 'em.

**Comment by tswast** [comment]
All docs links in the README are now updated.

File: README.md [hunk 1]
```diff
@@ -995,11 +995,11 @@ Apache 2.0 - See [LICENSE] for more information.
 [cloud-resourcemanager-docs]:https://cloud.google.com/resource-manager/
```
Comment (garrettjonesgoogle) [inline README.md]: This one needs `docs/` on the end

Comment (tswast) [inline README.md]: Done

**Pull request merged**